File: Python/Ex-1_Teams.py
# 
# Ex-1_Teams.py
# Created by Sebastian Jusca on 17/04/2023.
#

SUM = i = 0
print('Enter 15 numbers, it will display the sum of the odd ones\n')

while i < 15:
	n = int(input(''))
	if n % 2 != 0: SUM += n 
	i += 1

print(SUM)

File: C/Ex1_Teams.c
//
//  Ex1_Teams.c
//  Created by Sebastian Jusca on 22/09/2023.
//

void main(){
  int n_prog = 2, n_tring = 1, n;
  
  printf("Enter Value: \n");
  scanf("%d",&n);
  
  for(int i = 1; i < n; i++){
    n_tring += n_prog;
    n_prog++;
  }
  
  printf("%d\n", n_tring);
  
}